nsot/api/views.py

```diff
@@ -57,6 +58,24 @@ def not_found(self, pk=None, site_pk=None, msg=None):
         raise exc.NotFound(msg)
 
     def success(self, data, status=status.HTTP_200_OK, headers=None):
+        """
+        Return a positive API response.
+
+        :param data:
+            Serialized data
+
+        :param status:
+            (Optional) HTTP status code
+
+        :param headers:
+            (Optional) Dict of extra headers
+        """
+        if headers is None:
+            headers = self.kwargs.get('headers')
+
+        if status is None:
```

Comment:
I just realized that this would never actually do anything, because `status` wouldn't ever be `None` unless explicitly passed by the caller.